herbert_core/classes/instrument_classes/instrument/instruments/@IX_inst/IX_inst.m

```diff
@@ -287,6 +299,7 @@
         end
     end
     methods(Sealed)
+        %{
         function is = isempty(obj)
```

Comment:
as you introduced null_inst, this either should not here or always return false (or array of false, the size of obj -- certainly when heterogeneous is removed) 
You probably better remove this method, as default `isempty` would return `false` so no point of creating duplication. 

null_inst should overload `isempty` to `true`